### PR TITLE
Add guard for null organisations

### PR DESCRIPTION
**Pull request by holly-cummins**
When setting this up locally against a playground repository, I had the following exception: 

```
Exception: java.lang.NullPointerException: Cannot invoke "org.kohsuke.github.GHOrganization.getLogin()" because the return value of "org.kohsuke.github.GHEventPayload$PullRequest.getOrganization()" is null
at io.quarkus.bot.PushToProjects.pullRequestLabeled(PushToProjects.java:78)
at io.quarkus.bot.PushToProjects_Multiplexer_Subclass.pullRequestLabeled$$superforward1(Unknown Source)
at io.quarkus.bot.PushToProjects_Multiplexer_Subclass$$function$$9.apply(Unknown Source)
at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:53)
```

The docs at https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types say, for organisation, "The organization that was chosen by the actor to perform action that triggers the event.
The property appears in the event object only if it is applicable."

### CODE REVIEW

File: src/main/java/io/quarkus/bot/PushToProjects.java [hunk 1]
```diff
@@ -72,12 +72,17 @@ void pullRequestLabeled(@PullRequest.Labeled GHEventPayload.PullRequest pullRequ
             return;
         }
```
Comment (gsmet) [inline src/main/java/io/quarkus/bot/PushToProjects.java]: Thanks! Obviously, I did all my tests in an organization.

Please make it a condition here with a `return` so that the flow stays simple e.g.

```
if (pullRequestPayload.getOrganization() == null) {
    return;
}
```
I do that for all edge cases.

Also:
- no need to log something: it's just gonna pollute the log for no good reason
- we need to have the same thing for the issues above

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by holly-cummins** [comment]
Makes sense, will do, thank you!

**Review by gsmet — APPROVED** [review]
@holly-cummins I just did while I was at it.
